### ISSUE TITLE
Exception while rendering an SVG generated from a PDF in Xamarin.IOS

### ISSUE THREAD

**Issue by terrencejames** [luberda-molinet/FFImageLoading#1251]
### Description

I'm trying to render an SVG that I generated from a PDF in Xamarin.IOS. Simple SVGs such as icons work fine, but I run into exceptions when generating this particular SVG.

### Steps to Reproduce

I just load an SVG into a UIImageView like so:

```
            await ImageService.Instance.
                LoadFileFromApplicationBundle(@"MobileResources/Images/test.svg").
                WithCustomDataResolver(new SvgDataResolver()).
                IntoAsync(_imageView);
```

### Expected Behavior

SVG correctly renders without issue.

### Actual Behavior
Run into 
`system.ArgumentException: An item with the same key has already been added. Key: font-face
`
### Basic Information

I've attached the specific SVG as well here. I noticed that in the SVG string, there indeed are multiple keys for font-face...though I'm not sure how to solve this exception. Is my only workaround to load this in a browser?
[testPDF.zip](https://github.com/luberda-molinet/FFImageLoading/files/3109439/testPDF.zip)

**Comment by terrencejames** [comment]
[test.zip](https://github.com/luberda-molinet/FFImageLoading/files/3109492/test.zip)
A simpler test PDF seems to work. This one was generated using a different, free converter online. Could it be how the PDFs are converted into SVGs?

**Comment by terrencejames** [comment]
Actually, even though the above SVG doesnt run into any exceptions, it still has trouble rendering all the elements. I would like to know if this is just a limitation of the library (which is totally fine) for complex SVGs.

**Comment by daniel-luberda** [comment]
Hi @terrencejames, 

`text` tag is supported only partially (only simple scenarios would work), I'll need to work on that. The code is here: https://github.com/luberda-molinet/FFImageLoading/blob/master/source/FFImageLoading.Svg.Shared/SkSvg.cs#L665-L736

**Comment by daniel-luberda** [comment]
@terrencejames There was a lot of fixes / improvements in 2.4.6

Image renders correctly now:

![Screenshot 2019-04-30 at 11 58 01](https://user-images.githubusercontent.com/13569983/56954550-53875300-6b3f-11e9-868a-8086cc59bdd9.png)
